src/components/Carousel/index.js

```diff
@@ -211,11 +178,10 @@ export default class Carousel extends PureComponent {
                 slidesToShow={variableWidth ? 1 : adjustedShowCount}
                 infinite={loop}
                 draggable={false}
-                afterChange={this.handleAfterChange}
-                onInit={() => this.setState({ initialized: true })}
                 variableWidth={variableWidth}
-                lazyLoad='ondemand'
```

Comment:
was removing this intentional?

Reply:
sure wasn't

Reply:
ah, there was an issue with variablewidth and this working together.  but i forgot to add it back in conditionally